--- qutils.orig/vis/vis.mak	2020-07-12 18:13:11.000000000 +0100
+++ qutils/vis/vis.mak	2020-07-12 18:14:46.000000000 +0100
@@ -242,7 +242,7 @@
 ################################################################################
 # Begin Source File
 
-SOURCE=\quake\utils2\common\cmdlib.c
+SOURCE=..\common\cmdlib.c
 DEP_CPP_CMDLI=\
 	".\..\common\cmdlib.h"\
 	{$(INCLUDE)}"\sys\TYPES.H"\
@@ -257,7 +257,7 @@
 ################################################################################
 # Begin Source File
 
-SOURCE=\quake\utils2\common\bspfile.c
+SOURCE=..\common\bspfile.c
 DEP_CPP_BSPFI=\
 	".\..\common\cmdlib.h"\
 	".\..\common\mathlib.h"\
@@ -272,7 +272,7 @@
 ################################################################################
 # Begin Source File
 
-SOURCE=\quake\utils2\common\mathlib.c
+SOURCE=..\common\mathlib.c
 DEP_CPP_MATHL=\
 	".\..\common\cmdlib.h"\
 	".\..\common\mathlib.h"\
